File: tasks/translation/output/Qwen/Qwen2.5-32B-Instruct-AWQ/codenet/cot/Java/Python/s466963472_commented.py
import sys

def read():
    try:
        d = int(input())
        return d
    except ValueError:
        return None

def solve(d):
    ans = 0
    for i in range(1, 600 // d + 1):
        x = i * d
        y = x * x
        ans += y * d
    return ans

if __name__ == "__main__":
    while True:
        d = read()
        if d is None:
            break
        print(solve(d))
# End of Code

File: tasks/translation/output/Qwen/QwQ-32B-AWQ/codenet/direct/comment/comment/direct/Java/C/s466963472_commented.c
#include <stdio.h>

int d;

int solve() {
    int ans = 0;
    int i;
    for (i = 1; i * d < 600; i++) {
        int x = i * d;
        int y = x * x;
        ans += y * d;
    }
    return ans;
}

int main() {
    while (scanf("%d", &d) == 1) {
        printf("%d\n", solve());
    }
    return 0;
}
// End of Code
